public/docs/ts/latest/guide/architecture.jade

```diff
@@ -193,196 +224,208 @@ figure
   We add binding markup to the template HTML to tell Angular how to connect both sides.
-  
-  There are four forms of data binding syntax. Each form has a direction - to the DOM, from the DOM, or in both directions -
+
+  There are four forms of data binding syntax. Each form has a direction &mdash; to the DOM, from the DOM, or in both directions &mdash;
   as indicated by the arrows in the diagram.
 <br clear="all">
 :marked
-  We saw three forms of data binding in our [example](#template) template:
-+makeExample('architecture/ts/app/hero-list.component.1.html', 'binding', 'app/hero-list.component.html (excerpt)')(format=".")
+  We saw three forms of data binding in our [example](#templates) template:
+
++makeExcerpt('app/hero-list.component.1.html', 'binding')
+
 :marked
-  * The `{{hero.name}}` [*interpolation*](displaying-data.html#interpolation) 
-  displays the component's `hero.name` property value within the `<div>` tags.
-  
-  * The `[hero]` [*property binding*](template-syntax.html#property-binding) passes the `selectedHero` from
+  * The `{{hero.name}}` [*interpolation*](displaying-data.html#interpolation)
+  displays the component's `hero.name` property value within the `<li>` tags.
+
+  * The `[hero]` [*property binding*](template-syntax.html#property-binding) passes the value of `selectedHero` from
   the parent `HeroListComponent` to the `hero` property of the child `HeroDetailComponent`.
-  
-  * The `(click)` [*event binding*](user-input.html#click) calls the Component's `selectHero` method when the user clicks
-  on a hero's name
+
+  * The `(click)` [*event binding*](user-input.html#click) calls the component's `selectHero` method when the user clicks a hero's name.
 
   **Two-way data binding** is an important fourth form
-  that combines property and event binding in a single notation using the `ngModel` directive. 
-  We didn't have a two-way binding in the `HeroListComponent` template; 
-  here's an example from the `HeroDetailComponent` template (not shown):
+  that combines property and event binding in a single notation, using the `ngModel` directive.
+  We didn't have a two-way binding in the `HeroListComponent` template;
+  here's an example from the `HeroDetailComponent` template:
+
++makeExcerpt('app/hero-detail.component.html', 'ngModel')
 
-+makeExample('architecture/ts/app/hero-detail.component.html', 'ngModel')(format=".")
 :marked
   In two-way binding, a data property value flows to the input box from the component as with property binding.
   The user's changes also flow back to the component, resetting the property to the latest value,
   as with event binding.
-  
-  Angular processes *all* data bindings once per JavaScript event cycle, 
-  depth-first from the root of the application component tree. 
+
+  Angular processes *all* data bindings once per JavaScript event cycle,
+  depth-first from the root of the application component tree.
```

Comment:
Actually, this is wrong. Angular processes bindings from the root of the app down.  

I don't think we need to specify the details here. Let's just say something like:

>Angular processes *all* data bindings once per JavaScript event cycle from the root of the application component tree down through the branches to the leaves.